build.gradle

```diff
@@ -22,18 +22,17 @@ ext {
     minSdkVersion = 15
     compileSdkVersion = 28
     buildToolsVersion = '28.0.2'
-    supportLibVersion = '28.0.0'
```

Comment:
@MariusVolkhart I didn't replace this with `androidxLibVersion` because AndroidX libraries can be updated independently, but I'm not sure if that's what we want to do here or if we want to update them all at once. Let me know and I'll fix it

Reply:
Given that AndroidX version libraries independently, I think we should do so as well. So I agree with this change.